.github/ISSUE_TEMPLATE/pg-auto-failover-enterprise-release-checklist-template.md

```diff
@@ -0,0 +1,62 @@
+---
+name: pg-auto-failover-enterprise release checklist template
+about: Please make sure to follow the checklist below along the release process.
+title: Pg Auto Failover Enterprise release checklist - [vX.Y.Z]
+labels: release_checklist
+assignees: gurkanindibay
+
+---
+
+## Several things to note here
+
+These instructions assume you have `$VERSION`, `$PROJECT`, and `$REPO` environment variables set in your shell (e.g. `10.0.2`, `citus`, and `citus`). With those set, code from most steps can be copy-pasted.
+
+**After this checklist, you're still not done: open a release checklist for Enterprise too!**
+
+# Update OS Packages
+## Debian and RedHat
+- Change your directory to `packaging` repository directory & checkout `all-$PROJECT` branch.
```

Comment:
It does not make sense to have `$PROJECT` here, if you intend to create separate templates for each project.

Reply:
Fixed as you suggested